mods/default/craftitems.lua

```diff
@@ -100,7 +108,16 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 	if formname ~= "default:book" then return end
 	local player_name = player:get_player_name()
 	local inv = player:get_inventory()
-	local stack = player:get_wielded_item()
+	if not book_writers[player_name] then return end
```

Comment:
can you expand the `if` oneliners here? that'd help with seeing control flow at a glance